src/app/search-results/page.scss

```diff
@@ -0,0 +1,19 @@
+@import '../apps/page.scss';
+
+.search-page {
+    .section-header-title {
+        /* 
+            Overriding default styles from AppsPage.scss
+            Find a better way to extend/override these styles
+        */
+        font: var(--saf-type-body-default-lg-regular-standard) !important;
```

Comment:
I don't see any need for these "!important". I have made tests and they don't make any difference. Could you remove them?